src/components/Header/AppBanner.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {Icon, Segment} from 'semantic-ui-react';
+import {useQuery} from '@apollo/react-hooks';
+import {ALL_BANNERS} from '../../admin/queries';
+
+const AppBanner = () => {
+  const {loading, error, data} = useQuery(ALL_BANNERS, {
+    variables: {enabled: true},
+  });
+  const iconMap = {
+    INFO: 'info circle',
+    WARN: 'warning circle',
+    ERROR: 'times circle',
+  };
+
+  const isActive = (banner) => {
+    /* 
+    Banner is active if it is enabled && 
+    start date <= today && end date is after today 
+    */
+    if (!banner.startDate || !banner.endDate) {
```

Comment:
Does it make more sense to consider a banner active if it is enabled and both its start and end dates are null? I would immagine most banners would not have a known lifetime when they are created with the exception of scheduled maintenance messages.

Reply:
Hmm ok, I wasn't sure what kinds of messages we would be displaying. So a valid banner can be one without an end date